tests/helper.rb

```diff
@@ -13,6 +13,12 @@
   FOG_TESTING_TIMEOUT = ENV['FOG_TEST_TIMEOUT'] || 2000
   Fog.timeout = 2000
   Fog::Logger.warning "Setting default fog timeout to #{Fog.timeout} seconds"
+
+  # These sets of tests do not behave nicely when running mocked tests
+  Thread.current[:tags] << '-serverlove'
```

Comment:
It looks like since I originally wrote this the tests were removed. Which means this isn't much use.

I'll update accordingly.